src/js/compositions.js

```diff
@@ -0,0 +1,133 @@
+/**
+	Pullquote
+*/ 
+
+
+// Knight Lab Namespace
+KL = {};
+
+// Debug Mode
+KL.debug = true;
+
+
+/*	KL.Bind
+================================================== */
+KL.Bind = function (/*Function*/ fn, /*Object*/ obj) /*-> Object*/ {
+	return function () {
+		return fn.apply(obj, arguments);
+	};
+};
+
+/*	Required Files
+	Webpack
+	https://webpack.github.io/
+================================================== */
+
+// CORE
+KL.Util = 		require("core/KL.Util.js");
+KL.Class = 		require("core/KL.Class");
+KL.Events = 	require("core/KL.Events");
+KL.Browser = 	require("core/KL.Browser");
+
+// DOM
+KL.DomMixins = 	require("dom/KL.DomMixins");
+KL.Dom = 		require("dom/KL.Dom");
+KL.DomUtil = 	require("dom/KL.DomUtil");
+KL.DomEvent = 	require("dom/KL.DomEvent");
+
+// DATA
+KL.Data = 		require("data/KL.Data");
+
+// QUOTE
+KL.QuoteComposition = require("quote/KL.QuoteComposition");
+
+/*	Trace (console.log)
+	Wrapped in a function to allow a boolean switch
+	to show console log only if in debug mode. 
+================================================== */
+trace = function( msg ) {
+	if (KL.debug) {
+		if (window.console) {
+			console.log(msg);
+		} else if ( typeof( jsTrace ) != 'undefined' ) {
+			jsTrace.send( msg );
+		} else {
+			//alert(msg);
+		}
+	}
+}
+
+KL.Pullquote = (function() {
+
+	// DOM ELEMENTS
+	this.el = {
+		container: 			KL.Dom.get("pullquote-container"),
```

Comment:
I don't see why we can't just do `document.getElementById` instead of using the factory function `KL.DOM.get` which is doing the exact same thing